ci/install-upstream-wheels.sh

```diff
@@ -20,6 +20,7 @@ conda uninstall -y --force \
     bottleneck \
     sparse \
     flox \
+    datatree \
```

Comment:
it's probably fine as-is (and I'm always confused about the name), but should this be `xarray-datatree`, given that that's the package we are installing below?